src/GraphQLParser/AST/GraphQLVariableDefinition.cs

```diff
@@ -1,15 +1,20 @@
+using System.Collections.Generic;
+
 namespace GraphQLParser.AST
 {
-    public class GraphQLVariableDefinition : ASTNode
+    public class GraphQLVariableDefinition : ASTNode, IHasDirectivesNode
     {
-        public object? DefaultValue { get; set; }
```

Comment:
@Shane32 I somehow missed this `object` type before. Changed to `GraphQLValue` per the [spec](http://spec.graphql.org/October2021/#VariableDefinition).